src/test/compile-fail/regions-escape-loop-via-vec.rs

```diff
@@ -12,10 +12,18 @@
 fn broken() {
     let mut x = 3;
     let mut _y = vec!(&mut x);
+    //~^ NOTE reference must be valid for the block suffix following statement 1 at 14:30...
+    //~| NOTE borrow of `x` occurs here
+    //~| NOTE borrow of `x` occurs here
+    //~| NOTE borrow of `x` occurs here
```

Comment:
I had to add three of these in order to get this test to pass. It looks like this note appears three times in the json output for the same span. Is that a bug?